sovrin_client/test/anon_creds/test_public_repo.py

```diff
@@ -112,14 +114,16 @@ def testGetSchemaBySeqNo(submittedSchemaDefGvt, publicRepo, looper):
 
 
 def testGetSchemaByInvalidSeqNo(submittedSchemaDefGvt, publicRepo, looper):
```

Comment:
Do we have a test in CLI reproducing the issue from INDY-378? Does the CLI show exception properly in this case not crashing?